EIPS/eip-5585.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: 4966
+title: Authorization standard based on ERC721
+description: NFT owners can authorize other users to use their NFTs for business purpose.
+author: VeegaLabs Official (@VeegaLabs100), <kccinfo100@gmail.com>
+discussions-to: #4966

```

Comment:
Must be on FEM. https://ethereum-magicians.org/

Reply:
Thanks, @Pandapip1. I will make them happen and create a new pull request.